src/views/ProductInventory.vue

```diff
@@ -0,0 +1,342 @@
+          <ion-list>
+            <ion-list-header>{{ $t("Location") }}</ion-list-header>
+            <ion-item>
+              <ion-label>{{ $t("Product Store") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">Australia</ion-select-option>
+              </ion-select>
+            </ion-item>
+            <ion-item>
+              <ion-label>{{ $t("Facility") }}</ion-label>
+              <ion-select value="any">
+                <ion-select-option value="any">California Warehouse</ion-select-option>
+              </ion-select>
+            </ion-item>
+          </ion-list>
+        </div>
+        <div class="product-sort">
+          <ion-item lines="none" class="border-sort">
+            <ion-icon slot="start" :icon="folderOutline" />
+            <ion-label>{{ "Group by" }}</ion-label>
+            <ion-select value="any">
+              <ion-select-option value="any">Partent</ion-select-option>
+            </ion-select>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-icon slot="start" :icon="swapVerticalOutline" />
+            <ion-label>{{ $t("Sort") }}</ion-label>
+            <ion-select value="any">
+              <ion-select-option value="any">{{$t("Product name")}}</ion-select-option>
+            </ion-select>
+          </ion-item>
+        </div>
+        <div class="products">
+          <div class="product">
+            <div class="product-image">
+              <Image/>
+            </div>
+            <div class="mobileonly">
+              <div class="product-detail">
+                <div class="product-virtual">
+                  <ion-item>
+                    <ion-thumbnail slot="start" class="mobileonly-image">
+                      <Image/>
+                    </ion-thumbnail>
+                    <ion-label>
+                      <p>Brand</p>
+                      <h1>Virtual Name</h1>
+                      <p>{{ $t("Color") }} : color</p>
+                      <p>{{ $t("Size") }}: size</p>
+                    </ion-label>
+                  </ion-item>
+                </div>
+                <div class="product-tags">
+                  <ion-chip>
+                    <ion-icon/>
+                    <ion-label>Shopify ID</ion-label>
+                  </ion-chip>
+                </div>
+                <div class="product-metadata">
+                  <ion-item lines="none" detail>
+                    <ion-note slot="end" class="metatags">3 variants</ion-note>
+                  </ion-item>
+                </div>
+              </div>
+              <div class="desktop-only">
+                <ion-list>
+                  <ion-list-header>Variants</ion-list-header>
+
+                  <div class="product-detail"> 
+                    <div class="product-virtual">
+                      <ion-item lines="none">
+                        <ion-label>
+                          <ion-label>SKU</ion-label>
```

Comment:
Why ion-label is used inside ion-label ?

Reply:
done

Reply:
done